project/src/test/java/com/google/sps/UserProfileEventsTest.java

```diff
@@ -179,4 +177,39 @@ public void verifyGetUserEvents_notLoggedIn_redirectToLogin() throws Exception {
     Mockito.verify(response).sendRedirect("/index.html");
   }
 
+  /*Test to get all events logged in user is participating in */
+  @Test
+  public void getUserEventsParticipating() throws IOException {
+    //Logged in user
+    User user = TestUtils.newUserWithEmail(EMAIL);
+    String userEmail = user.getEmail(); 
+    SpannerTasks.insertOrUpdateUser(user);
+    authenticationHelper
```

Comment:
I'm writing a helper function called `setAuthenticationHelper()` for this part in my eventsHosting PR, so you can call it instead